selendroid-standalone/src/main/java/io/selendroid/standalone/server/handler/NetworkConnectionHandler.java

```diff
@@ -78,9 +78,14 @@ public Response handleRequest(HttpRequest request, JSONObject payload) throws JS
 
 
     // nice and magical, this will 'close' the settings view and reopen the last application, the one under test! :)
-    device.inputKeyevent(KeyEvent.KEYCODE_DPAD_UP);
-    device.inputKeyevent(KeyEvent.KEYCODE_DPAD_CENTER);
-
+    if (deviceAPILevel == 21) {
```

Comment:
I updated the if statement to >= 21. 

3 back keys is the magical number here to get back to the app under test. There is no one action to take that will get you back to the app that I know of. The order of pages the process goes through is this.
App > Settings > network settings.